ksqldb-parser/src/main/java/io/confluent/ksql/parser/tree/CreateConnector.java

```diff
@@ -34,32 +34,41 @@
   private final String name;
   private final ImmutableMap<String, Literal> config;
   private final Type type;
+  private final boolean notExists;
```

Comment:
Needs adding to `hashCode`, `equals` and `toString`.

Reply:
Fixed this